### PR TITLE
Updating css-style testing docs to mention StoryShots image snapshots feature

### PR DESCRIPTION
**Pull request by anescobar1991**
Issue: With https://github.com/storybooks/storybook/pull/2413 being merged into master and adding image snapshot capabilities into StoryShots the [documentation](https://storybook.js.org/testing/css-style-testing/#supported-cssstyle-testing-frameworks) should reflect the change.

## What I did

Updated css-style-testing doc to mention StoryShots integration with `jest-image-snapshot`

## How to test

View doc in markdown editor.

### CODE REVIEW

File: docs/src/pages/testing/css-style-testing/index.md [hunk 1]
```diff
@@ -23,10 +23,4 @@ Then, we can see the above story using the following URL:
 
 Just like that, you can access all of the stories in your Storybook.
 
-## Supported CSS/Style Testing Frameworks
-
-It will be hard to use all the frameworks we've [mentioned](/testing/react-ui-testing#3-css-style-testing), but we'll be able to use frameworks which are based on URL as the input source. (Such as [BackstopJS](https://github.com/garris/BackstopJS) and [Gemini](https://github.com/gemini-testing/gemini))
-
-- [Loki](https://loki.js.org/): Is a great tool for visual regression testing.
```
Comment (Hypnosphi) [inline docs/src/pages/testing/css-style-testing/index.md]: I think all those tools are still worth mentioning

**Review by Hypnosphi — CHANGES_REQUESTED** [review]

**Review by Hypnosphi — APPROVED** [review]

**Comment by Hypnosphi** [comment]
@tmeasday would you like Chromatic to be mentioned here as well?

**Review by ndelangen — APPROVED** [review]

**Comment by tmeasday** [comment]
@Hypnosphi sure! We should mention the other commercial offerings that support storybook too: [Percy](https://percy.io/docs/clients/javascript/react-storybook) and [Screener](https://screener.io/v2/docs).

**Comment by anescobar1991** [comment]
@Hypnosphi should I do that here? Or should those be added in a separate PR?

**Comment by Hypnosphi** [comment]
@anescobar1991 as you wish =)

**Comment by anescobar1991** [comment]
@Hypnosphi I added them in 728cad8e1975eaf82d03f55c0df0416fe00bd9a7 :)

File: docs/src/pages/testing/css-style-testing/index.md [hunk 2]
```diff
@@ -21,12 +21,14 @@ Then, we can see the above story using the following URL:
...
+Just like that, you can access all of the stories in your Storybook and use them as a base for visual regression testing.
 
 ## Supported CSS/Style Testing Frameworks
 
-It will be hard to use all the frameworks we've [mentioned](/testing/react-ui-testing#3-css-style-testing), but we'll be able to use frameworks which are based on URL as the input source. (Such as [BackstopJS](https://github.com/garris/BackstopJS) and [Gemini](https://github.com/gemini-testing/gemini))
+There are several frameworks that have out of the box Storybook integrations:
+- [StoryShots](https://github.com/storybooks/storybook/tree/master/addons/storyshots) with its [seamless integration](https://github.com/storybooks/storybook/tree/master/addons/storyshots#configure-storyshots-for-image-snapshots) with [jest-image-snapshot](https://github.com/americanexpress/jest-image-snapshot)
+- [Loki](https://loki.js.org/)
+- [Perci](https://percy.io/docs/clients/javascript/react-storybook)
+- [Screener](https://screener.io/v2/docs)
```
Comment (Hypnosphi) [inline docs/src/pages/testing/css-style-testing/index.md]: And also [Chromatic](https://www.chromaticqa.com/)

**Pull request merged**